docs/docs/databases/installing-database-drivers.mdx

```diff
@@ -22,8 +22,8 @@ A list of some of the recommended packages.
 
 | Database                                                  | PyPI package                                                                       | Connection String                                                                                           |
 | --------------------------------------------------------- | ---------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------- |
-| [Amazon Athena](/docs/databases/athena)                   | `pip install "PyAthenaJDBC>1.0.9` , `pip install "PyAthena>1.2.0`                  | `awsathena+rest://{aws_access_key_id}:{aws_secret_access_key}@athena.{region_name}.amazonaws.com/{ `        |
-| [Amazon DynamoDB](/docs/databases/dynamodb)               | `pip install "PyDynamoDB>=0.4.2`                                                   | `dynamodb://{access_key_id}:{secret_access_key}@dynamodb.{region_name}.amazonaws.com?connector=superset`    |
+| [Amazon Athena](/docs/databases/athena)                   | `pip install "PyAthenaJDBC>1.0.9"` , `pip install "pyathena[pandas]>=2,<3"`              | `awsathena+rest://{aws_access_key_id}:{aws_secret_access_key}@athena.{region_name}.amazonaws.com/{ `        |
```

Comment:
I believe it would be better if we treat this table as an example and point the reader to the files where the most up to date versions can be found such as `setup.py` and `requirements/*`. That way, this documentation will be more resilient to version upgrades.

Reply:
Agree, should I extend the scope of the current PR, or open a new PR for this?

Reply:
@michael-s-molina 
I tried looking into this issue once before. I couldn't figure out why this table had been changed from referencing the optional extras requirements that are managed in the setup.py to the state the table now is in where it no longer specifies the optional extras requirements but instead directly lists the actual driver package names in the documentation table itself.

I think there is a bigger question of how to manage the driver compatibility and support in general. I.e. should the be such a concept of "officially supported and tested drivers" that should have an expectation of always working, vs community-supported and tested drivers?